### DISCUSSION TITLE
How to change title(titlebar)'s theme

### DISCUSSION THREAD

**Discussion by bingryan** [tauri-apps/tauri#6710] [Q&A] [answered]
the app theme is depend on the system settings, such as: dark | light | auto
If our system theme is light, but I want my app theme to be dark.
After passing `body.classList.add('dark');`, the theme of the `app title` will not be changed to dark(main content change to dark). Is there a way to change the theme of title(titlebar)

**Comment by FabianLars** [comment, marked as answer]
This is currently not possible but this is the relevant feature request: https://github.com/tauri-apps/tauri/issues/5279
